src/main/java/com/squareup/catalog/demo/util/CatalogObjectTypes.java

```diff
@@ -0,0 +1,28 @@
+package com.squareup.catalog.demo.util;
+
+import com.squareup.square.models.CatalogObject;
+
+/**
+ * Utility enum to capture different {@link CatalogObject} types.
+ */
+public enum CatalogObjectTypes {
+  ITEM("ITEM"),
```

Comment:
You can just write these as 
```
public enum CatalogObjectTypes {
  ITEM,
  ITEM_VARIATION,
  ETC
}
```
By default toString() and the name() methods will return the strings: "ITEM" and "ITEM_VARIATION"

Reply:
Nice, didn't know that. Thanks